Form/Type/ColorSelectorType.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Sonata package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ */
+
+namespace Sonata\CoreBundle\Form\Type;
+
+use Sonata\CoreBundle\Color\Colors;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+class ColorSelectorType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @todo Remove it when bumping requirements to SF 2.7+
+     */
+    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    {
+        $this->configureOptions($resolver);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults(array(
+            'choices'            => Colors::getAll(),
```

Comment:
Would be more user friendly to use [`preferred_choices`](http://symfony.com/doc/2.3/reference/forms/types/choice.html#preferred-choices) options with basic colors like just `red`, `blue`, `green`, `yellow`, `black` etc...

What do you think?